core/src/main/kotlin/com/xorker/draw/mafia/MafiaGameService.kt

```diff
@@ -61,34 +60,32 @@ internal class MafiaGameService(
         }
     }
 
-    override fun voteMafia(session: Session, targetUserId: UserId) {
-        val gameInfo = session.getGameInfo()
-
-        val voter = session.user
+    override fun voteMafia(user: User, targetUserId: UserId) {
+        val gameInfo = mafiaGameRepository.getGameInfo(user.id) ?: throw InvalidRequestValueException
```

Comment:
P3
```suggestion
        val gameInfo = user.getGameInfo()
```
메서드 활용하면 좋을 것 같습니다.